sceneview/src/main/java/io/github/sceneview/node/CubeNode.kt

```diff
@@ -48,6 +44,26 @@ open class CubeNode(
     parent = parent,
     renderableApply = renderableApply
 ) {
+
+    constructor(
+        engine: Engine,
+        size: Size = Cube.DEFAULT_SIZE,
+        center: Position = Cube.DEFAULT_CENTER,
+        /**
+         * Binds a material instance to all primitives.
+         */
+        materialInstance: MaterialInstance,
```

Comment:
Should also be nullable for every geometries.

Reply:
fixed